frontend/lib/js/form-components/ClearableInput.js

```diff
@@ -2,28 +2,53 @@
 
 import React                from 'react';
 import T                    from 'i18n-react';
+import NumberFormat         from 'react-number-format';
 
 import { isEmpty }          from '../common/helpers';
+import { MONEY_RANGE }      from '.';
```

Comment:
Maybe a question of taste -- `import {} from '.'` is not very descriptive, we could also import this directly from `'./filterTypes'`